index.md

```diff
@@ -48,6 +48,361 @@ def is_even(num):
     return num % 2 == 0
 ```
 
+### E1101: no-member {#E1101}
+
+We expect you to access an object(variable, function...) for an existent member.
+For example, you should not call a variable or function which is not defined.
+In the following example, the Rectangle class does not have the area()
+method(member).
+
+
+```python
+class Rectangle:
+    def __init__(self, width, height):
+        """
+        Initialize the Rectangle.
+
+        @type self: Rectangle
+        @type width: int
+        @type height: int
+        @rtype: None
+        """
+        self.width = width
+        self.height = height
+
+r = Rectangle(5, 4)
+
+# no such member in Rectangle class
+print(r.area())  # Error on this line
+```
+
+### E1102: not-callable {#E1102}
+
+We expect you to call an object only if it has been inferred to a callable object.
```

Comment:
Don't use the phrase "callable object." Just say you can only use the syntax for calling a function on variables that are actually functions. Make sure you explain what the syntax for calling a function is, however.